cmds/webboot/webboot_test.go

```diff
@@ -145,3 +146,35 @@ func TestDirOption(t *testing.T) {
 		}
 	}
 }
+
+// This test may not work for every one.
+// because the test need a usb stick that contains a /Image folder
+// which structure is
+// +-- Image
+// |  +-- tinycore1
+// |  |  +-- TinyCorePure64.iso
+// |  |  +-- TinyCorePure64 (copy).iso
+// |  +-- tinycore2
+// |     +-- TinyCorePure64-10.1.iso
+// |     +-- TinyCorePure64-10.1 (copy).iso
+// ...
+func TestGetCachedDirectory(t *testing.T) {
+	mp, err := getCachedDirectory()
+	if err != nil {
+		t.Fatalf("Fail to find the USB stick: %+v", err)
+	}
+	if mp == nil {
```

Comment:
getCachedDirectory should always return nil if err is != nil?

Reply:
Yes, the function returns both mount point  and error. and if there is a error, function always return nil for the mount point.

Reply:
Oh I think I got you do you mean I should not check mp == nil here because if there is a error the mp will be nil too? I think if there is a error the t.Fatalf("Fail to find the USB stick: %+v", err) will be call and the test will exit immediately? but I will fix that thanks!